Detectors/DCS/testWorkflow/dcs-data-workflow.cxx

```diff
@@ -0,0 +1,43 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DetectorsDCS/DataPointIdentifier.h"
+#include "DetectorsDCS/DataPointValue.h"
+#include "Framework/TypeTraits.h"
```

Comment:
The CI wants you to ``#include <unordered_map>`` here

Reply:
Hello @shahor02 , It is included. I can move it above, let's see if it makes it happy. I will do it when ready with few minor changes.

Reply:
Let's see further tests...